src/harness/fourslash.ts

```diff
@@ -1656,12 +1656,31 @@ Actual: ${stringify(fullActual)}`);
                 this.raiseError("No emitThisFile is specified in the test file");
             }
 
+            return emitFiles;
+        }
+
+        public verifyGetEmitOutput(expectedOutputFiles: ReadonlyArray<string>): void {
+            const outputFiles = ts.flatMap(this.getEmitFiles(), e => this.languageService.getEmitOutput(e.fileName).outputFiles);
+
+            assert.deepEqual(outputFiles.map(f => f.name), expectedOutputFiles);
+
+            for (const { name, text } of outputFiles) {
+                const fromTestFile = this.getFileContent(name);
+                if (fromTestFile !== text) {
+                    this.raiseError("Emit output is not as expected: " + showTextDiff(fromTestFile, text));
+                }
+            }
+        }
+
+        public baselineGetEmitOutput(insertResultsIntoVfs?: boolean): void {

```

Comment:
The `insertResultsIntoVfs` flag only existed to make writing `fourslash` tests with declaration maps easier. If they're all in the `server` folder now, and none of them use `baselineGetEmitOutput`, then this flag (and the associate branch of code) is unused and aught to be removed.